Document/0x05b-Basic-Security_Testing.md

```diff
@@ -752,16 +752,49 @@ Check the [Termux Wiki](https://wiki.termux.com/wiki/Remote_Access "Termux Remot
 
 #### Obtaining and Extracting Apps
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1238>
+There are several ways of extracting apk files from a device. You will need to decide which one is the easiest method depending if the app is public or private.
 
 ##### App Store
```

Comment:
The app store, kinda feels like it is about the google play store. Maybe we call this alternative app-stores in the title?

Reply:
Changed it 'Alternative App Stores'